tests/Fixture/ArticlesFixture.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace FOC\Search\Test\Fixture;
+
+use Cake\TestSuite\Fixture\TestFixture;
+
+class ArticlesFixture extends TestFixture {
+
+/**
+ * Fields
+ *
+ * @var array
+ */
+    public $fields = [
+        'id' => ['type' => 'string', 'null' => false, 'length' => 36],
+        'title' => ['type' => 'string', 'null' => false, 'default' => null],
+        'created' => ['type' => 'datetime', 'null' => true, 'default' => null],
+        'modified' => ['type' => 'datetime', 'null' => true, 'default' => null],
+        '_constraints' => [
+            'primary' => ['type' => 'primary', 'columns' => ['id']],
+        ]
+    ];
+
+/**
+ * Records
+ *
+ * @var array
+ */
+    public $records = [
+        [
+            'id' => '1',
```

Comment:
I'm getting errors when I remove it. No matter if I remove it from the record array or the schema declaration.